qa/apache2/qa.php.net.conf

```diff
@@ -38,10 +46,16 @@
         Require all denied
     </FilesMatch>
 
-    CustomLog ${APACHE_LOG_DIR}/qa_access.log combined
-    ErrorLog ${APACHE_LOG_DIR}/qa_error.log
+    CustomLog ${APACHE_LOG_DIR}/qa_ssl_access.log combined
+    ErrorLog ${APACHE_LOG_DIR}/qa_ssl_error.log
```

Comment:
Do we really need `ssl` in the name?

Reply:
No, it was just how it was configured now (and there were two sets of log files for http and https). I've made it more consistent between qa.php.net and news-web.php.net now.